frontend/src/modules/organization/components/view/organization-view-header.vue

```diff
@@ -229,18 +187,75 @@ const formattedInformation = (value, type) => {
   // Render inforamation depending on type
   if (type === 'date') {
     return formatDate({ timestamp: value });
-  } if (type === 'number') {
+  }
+  if (type === 'number') {
     return formatNumber(value);
-  } if (type === 'relative') {
+  }
+  if (type === 'relative') {
     return formatDateToTimeAgo(value);
-  } if (type === 'compact') {
+  }
+  if (type === 'compact') {
     return formatNumberToCompact(value);
-  } if (type === 'revenueRange') {
+  }
+  if (type === 'revenueRange') {
     return formatRevenueRange(value);
   }
 
   return value;
 };
+
+const fetchMembers = async () => {
+  const filterToApply = {
+    organizations: [props.organization.id],
+  };
+
+  if (query.value && query.value !== '') {
+    filterToApply.or = [
+      {
+        name: {
+          textContains: query.value,
+        },
+      },
+      {
+        bio: {
+          textContains: query.value,
+        },
+      },
+      {
+        email: {
+          textContains: query.value,
+        },
+      },
+    ];
+  }
+
+  const { data } = await authAxios.post(
```

Comment:
Please don't fetch requests directly here. Use MemberService.list to do this request